kotlin-web-spring-boot-3/src/main/kotlin/com/vndevteam/kotlinwebspringboot3/application/logging/CustomResponseBody.kt

```diff
@@ -0,0 +1,83 @@
+package com.vndevteam.kotlinwebspringboot3.application.logging
+
+import com.vndevteam.kotlinwebspringboot3.util.MapperUtils
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.core.MethodParameter
+import org.springframework.http.MediaType
+import org.springframework.http.converter.HttpMessageConverter
+import org.springframework.http.server.ServerHttpRequest
+import org.springframework.http.server.ServerHttpResponse
+import org.springframework.http.server.ServletServerHttpRequest
+import org.springframework.http.server.ServletServerHttpResponse
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.servlet.mvc.method.annotation.ResponseBodyAdvice
+import java.sql.Timestamp
+
+@ControllerAdvice
+class CustomResponseBody : ResponseBodyAdvice<Any> {
+    companion object {
+        val log: Logger = LoggerFactory.getLogger(CustomResponseBody::class.java)
+    }
+    
+    override fun supports(returnType: MethodParameter, converterType: Class<out HttpMessageConverter<*>>): Boolean {
+        return true
+    }
+
+    override fun beforeBodyWrite(
+        body: Any?,
+        returnType: MethodParameter,
+        selectedContentType: MediaType,
+        selectedConverterType: Class<out HttpMessageConverter<*>>,
+        request: ServerHttpRequest,
+        response: ServerHttpResponse
+    ): Any? {
+        if (request is ServletServerHttpRequest && response is ServletServerHttpResponse) {
+            logResponse(
+                request.servletRequest,
+                response.servletResponse, body
+            )
+        }
+
+        return body
+    }
+
+    private fun logResponse(
+        httpServletRequest: HttpServletRequest,
+        httpServletResponse: HttpServletResponse,
```

Comment:
```suggestion
        req: HttpServletRequest,
        res: HttpServletResponse,
```
@haiks-2382 Như này cho gọn code hí